packages/esm-home-app/src/default-dashboard-redirect.component.tsx

```diff
@@ -16,5 +16,9 @@ export function DefaultDashboardRedirect() {
   const dashboards = ungroupedDashboards as Array<DashboardConfig>;
   const activeDashboard = dashboards.find((dashboard) => dashboard.name === defaultDashboard);
 
+  if (!activeDashboard && dashboards.length > 0) {
+    return <Navigate to={`/home/${dashboards[0].name}`} />;
+  }
+
   return <Navigate to={`/home/${activeDashboard.name}`} />;
```

Comment:
So what happens if there's no active dashboard and no dashboards available? Won't that still leave us in the same place?